index.css

```diff
@@ -0,0 +1,216 @@
+* {
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+}
+
+html,
+body {
+    height: 100%;
+    font-size: 14px;
+    font-family: arial, sans-serif;
+}
+
+.main {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    width: 100%;
+    background-color: white;
```

Comment:
prefer to use hex or rgb colors value, different browser use different shades for color values, hence it is a safer choice to use hex or rgb.